machine.go

```diff
@@ -34,6 +35,7 @@ type machine struct {
 	powerState  string
 
 	// NOTE: consider some form of status struct
+	status        string
```

Comment:
If the schema value has it as an integer, why are we having it stored and returned as a string?

Reply:
This is because the `NodeStatus*` values are defined as a string.

https://github.com/juju/gomaasapi/blob/aa561369c76791f1ab2591e5c8981ae10b9bb4f5/enum.go#L9-L13

If we can change the `NodeStatus*` values to integer, this method can return an integer.

Reply:
@howbazaar I've changed the returned value to int from string at https://github.com/juju/gomaasapi/commit/a51e019841659bcc2809587b3b1cec4c455b6e58. Could you please check it?

Reply:
ping @howbazaar